src/cryptoadvance/specter/services/service_manager.py

```diff
@@ -0,0 +1,163 @@
+        try:
+            module = import_module(
+                f"cryptoadvance.specter.services.{service_id}.config"
+            )
+        except ModuleNotFoundError:
+            logger.warning(
+                f"Service {service_id} does not have a service Configuration! Skipping!"
+            )
+            return
+        main_config_clazz_name = app.config.get("SPECTER_CONFIGURATION_CLASS_FULLNAME")
+        main_config_clazz_slug = main_config_clazz_name.split(".")[-1]
+        potential_config_classes = []
+        for attribute_name in dir(module):
+            attribute = getattr(module, attribute_name)
+            if isclass(attribute):
+                clazz = attribute
+                potential_config_classes.append(clazz)
+                if clazz.__name__.endswith(
+                    main_config_clazz_slug
+                ):  # e.g. BaseConfig or DevelopmentConfig
+                    cls.import_config(clazz)
+                    return
+
+        logger.warning(
+            f"Could not find a configuration for Service {module} ... trying parent-classes of main-config"
+        )
+        config_module = import_module(".".join(main_config_clazz_name.split(".")[0:-1]))
+
+        config_clazz = getattr(config_module, main_config_clazz_slug)
+        config_candidate_class = config_clazz.__bases__[0]
+        while config_candidate_class != object:
+            for clazz in potential_config_classes:
+                if clazz.__name__.endswith(config_candidate_class.__name__):
+                    cls.import_config(clazz)
+                    return
+            config_candidate_class = config_candidate_class.__bases__[0]
+
+    @classmethod
+    def import_config(cls, clazz):
+        logger.info(f"Loading Service-specific configuration from {clazz}")
+        for key in dir(clazz):
+            if key.isupper():
+                if app.config.get(key):
+                    raise Exception(
+                        f"Config {clazz} tries to override existing key {key}"
+                    )
+                app.config[key] = getattr(clazz, key)
+                logger.debug(f"setting {key} = {app.config[key]}")
+
+    @property
+    def services(self):
+        return self._services
+
+    @property
+    def services_sorted(self):
+        service_names = sorted(
+            self._services, key=lambda s: self._services[s].sort_priority
+        )
+        return [self._services[s] for s in service_names]
+
+    def set_active_services(self, service_names_active):
+        logger.debug(f"Setting these services active: {service_names_active}")
+        self.specter.update_services(service_names_active)
+        for _, service in self.services.items():
+            logger.debug(
+                f"Setting service '{service.id}' active to {service.id in service_names_active}"
+            )
+            service.active = service.id in service_names_active
```

Comment:
I think this is no longer needed. The call of this in the service_endpoint is also no longer needed.